mavlink/definitions/uavionix.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version='1.0'?>
+<mavlink>
+    <include>common.xml</include>
+    <enums>
+      <enum name="UAVIONIX_SDA_LEVEL">
```

Comment:
@brandon-hass I saw the other dialects doing prefixes like that, and the MAVLink common message set enums all start with MAVLink, so I thought that was the standard.